src/static/css/index.css

```diff
@@ -554,27 +520,6 @@ p {
     justify-content: center;
   }
 
-  .intro-year {
-    display: block;
-    margin: 0;
-    width: auto;
-    color: #f7f779;
-    font-size: 40vw;
-    line-height: 100%;
-    font-weight: bold;
-  }
-
-  .intro-image-wrapper {
-    display: block;
-    max-width: 100%;
-    margin-top: 60px;
-    margin-right: -46px;
```

Comment:
By removing this -46px we get a change on mobile as this was there to offset the fact that the main body of the image is not in the centre of the image so looks weird against the above button:

Old:

![Image is aligned under Start Exploring button](https://user-images.githubusercontent.com/10931297/86537005-ef4ee480-bee3-11ea-8d76-689221fe6ac9.png)

New:

![Image is not aligned under Start Exploring button](https://user-images.githubusercontent.com/10931297/86537113-9764ad80-bee4-11ea-92b1-518549181756.png)

It's a bit of a hack and a magic number to be honest and maybe the better solution it to change the image to make it centrally aligned?

Reply:
I pushed another commit to revert this pseudo center alignment for this image.

Reply:
Don’t think we should just widen the actually image file and remove this hack? This [caught me out elsewhere too btw](https://github.com/HTTPArchive/almanac.httparchive.org/pull/833#discussion_r428075979) and really think we should just fix the image if we want it “cantered” all the time.

Reply:
Will have a look at it.

Reply:
I managed to push something on this matter. Let me know what you think @bazzadp

Reply:
Thanks for that. Much better I think! Resolving.